ports/psoc6/mpconfigport.h

```diff
@@ -107,21 +108,64 @@
 #define MICROPY_FATFS_LFN_CODE_PAGE             437 /* 1=SFN/ANSI 437=LFN/U.S.(OEM) */
 #define MICROPY_FATFS_RPATH                     (2)
 
+// set to 1 to enable filesystem to be loaded on external qspi flash
+// if set to 0, filesystem is located in an allotted area of internal flash of PSoC6
+#define MICROPY_ENABLE_EXT_QSPI_FLASH               (1)
+
+#ifndef MICROPY_BOARD_ENTER_BOOTLOADER
+#define MICROPY_BOARD_ENTER_BOOTLOADER(nargs, args)
+#endif
+
+#define MICROPY_TRACKED_ALLOC        (MICROPY_SSL_MBEDTLS)
+
+#define MICROPY_PY_UCRYPTOLIB        (1)
+#define MICROPY_PY_UCRYPTOLIB_CTR    (1)
+#define MICROPY_PY_UCRYPTOLIB_CONSTS (1)
+
+#define MICROPY_PY_UHASHLIB          (1)
+#define MICROPY_PY_UHASHLIB_MD5      (1)
+#define MICROPY_PY_UHASHLIB_SHA1     (1)
+#define MICROPY_PY_UHASHLIB_SHA256   (1)
+
+#define MP_STATE_PORT MP_STATE_VM
+
+// // By default networking should include sockets, ssl, websockets, webrepl
+// #if MICROPY_PY_NETWORK
 // By default networking should include sockets, ssl, websockets, webrepl, dupterm.
-// #define MICROPY_PY_NETWORK                      (1)
-// #define MICROPY_PY_USOCKET                      (1)
-// #define MICROPY_PY_UWEBSOCKET                   (1)
-// #define MICROPY_PY_WEBREPL                      (1)
-#define MICROPY_PY_OS_DUPTERM                      (1)
+#define MICROPY_PY_NETWORK                      (1)
+#define MICROPY_PY_USOCKET                      (1)
+#define MICROPY_PY_UWEBSOCKET                   (1)
+#define MICROPY_PY_WEBREPL                      (1)
+#define MICROPY_PY_OS_DUPTERM                   (1)
+
+
+// #ifndef MICROPY_PY_USSL
+// #define MICROPY_PY_USSL                 (1)
+// #endif
+
+// #ifndef MICROPY_PY_UWEBSOCKET
+// #define MICROPY_PY_UWEBSOCKET           (1)
+// #endif
+
+// #ifndef MICROPY_PY_UHASHLIB_SHA1
+// #define MICROPY_PY_UHASHLIB_SHA1        (0)
+// #endif
+
+// #ifndef MICROPY_PY_WEBREPL
+// #define MICROPY_PY_WEBREPL              (1)
+// #endif
 
-#define MP_STATE_PORT                           MP_STATE_VM
+// #ifndef MICROPY_PY_WEBREPL
+// #define MICROPY_PY_WEBREPL              (1)
+// #endif
 
-#define MICROPY_PY_UCRYPTOLIB                   (1)
-#define MICROPY_PY_UCRYPTOLIB_CTR               (1)
-#define MICROPY_PY_UCRYPTOLIB_CONSTS            (1)
+extern const struct _mp_obj_type_t mp_network_ifx_whd_type;
+#define MICROPY_HW_NIC_IFX_WHD  \
```

Comment:
I thought as per convention, we only defined and mapped `QSTRs` inside module and class files. Any other ports defining QSTRs outside these files, for example, in `mpconfigport.h` etc